engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/sql/SqlExpressionParser.scala

```diff
@@ -116,7 +116,10 @@ class SqlExpression(private[sql] val columnModels: Map[String, ColumnModel],
 }
 
 // empty for now
-case object SqlExpressionTypingInfo extends ExpressionTypingInfo
+case object SqlExpressionTypingInfo extends ExpressionTypingInfo {
+
+  override def typingResult: TypingResult = typing.Unknown
```

Comment:
I think we can safely change it to case class with typingResult - we use it in place where it's already determined

Reply:
👍